tools/ci_connector_ops/ci_connector_ops/pipelines/actions/environments.py

```diff
@@ -301,7 +301,8 @@ async def with_ci_connector_ops(context: PipelineContext) -> Container:
     """
     python_base_environment: Container = with_python_base(context, "python:3-alpine")
     python_with_git = with_alpine_packages(python_base_environment, ["gcc", "libffi-dev", "musl-dev", "git"])
-    return await with_installed_python_package(context, python_with_git, CI_CONNECTOR_OPS_SOURCE_PATH, exclude=["pipelines"])
+
+    return await with_installed_python_package(context, python_with_git, CI_CONNECTOR_OPS_SOURCE_PATH)
```

Comment:
The pipelines folder is now excluded by default from all operations mounting the repo to container filesystem.